### ISSUE TITLE
License clarification

### ISSUE THREAD

**Issue by gavinhenderson** [antijingoist/opendyslexic#74]
Sorry this is not a feature request but question about the license.

[I am building an Open Source (MIT) application ](https://github.com/AceCentre/aac-launchpad) which is a templating system which generates PDF files. A user answers a set of questions and in turn a PDF is generated to match their specification. The application has multiple interfaces. It can be used via CLI, API and a web app.

I am very keen to allow people to chose `opendyslexic` as the font to be used in their generated PDFs. To do this I would use the font in the backend service, which would bundle the font as part of the PDF. A user of my software may never download anything other than a PDF.

My questions about how the license will impact my work are:
* Am I *required* to give attributions to the font?
* If I am required, where am I required to give the attribution? In every PDF? In my Github README? In my web interface?
* Am I required to include a copy of the license?
* Can I distribute my software under any license?
* Can I distribute my software for profit (not currently my plan but just want to cover my bases)

I have read both the `OFL.txt` file and the `OFL-FAQ.txt` file in depth. However, I was unable to answer my questions based on these documents, mainly because I don't understand the legal wording.

Related issue: https://github.com/AceCentre/aac-launchpad/issues/2

**Comment by cloudmaker97** [comment]
Yes, a license is pretty much appreciated and could only be legal set by @antijingoist , so i can't create a PR for that. 😟

**Comment by antijingoist** [comment]
opendyslexic is SIL-OFL and included in the repo. 
SIL-OFL does not affect your software license. The FAQ is fairly transparent.
